src/python/nimbusml/linear_model/linearsvmbinaryclassifier.py

```diff
@@ -0,0 +1,183 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------------------------
+# - Generated by tools/entrypoint_compiler.py: do not edit by hand
+"""
+LinearSvmBinaryClassifier
+"""
+
+__all__ = ["LinearSvmBinaryClassifier"]
+
+
+from sklearn.base import ClassifierMixin
+
+from ..base_predictor import BasePredictor
+from ..internal.core.linear_model.linearsvmbinaryclassifier import \
+    LinearSvmBinaryClassifier as core
+from ..internal.utils.utils import trace
+
+
+class LinearSvmBinaryClassifier(
+        core,
```

Comment:
Pls add this classifier to test_estimator_checks.py

Reply:
@ganik I verified that LinearSvmBinaryClassifier checks are being executed in test_estimator_checks.py. There are no errors so no changes need to be made to test_estimator_checks.py.

Reply:
Great!

---
In reply to: [302703328](https://github.com/microsoft/NimbusML/pull/180#discussion_r302703328) [](ancestors = 302703328)